### DISCUSSION TITLE
Highlight search term in the buffer previewer

### DISCUSSION THREAD

**Discussion by savchenko** [linrongbin16/fzfx.nvim#607] [Ideas]
Unsure is this is possible, but I thought it would help to highlight the current search term.

See how on the left `require` is highlighted with red?

Having the same on the right would help user to pinpoint the exact location within the file.

![image](https://github.com/linrongbin16/fzfx.nvim/assets/300146/e04af938-2efa-4834-830a-935b78e0e60b)

**Comment by linrongbin16** [comment]
It is not eady to do so.

Because it uses different technology:
highlight on the left is using the fzf 'fg+' colors, the right side is using bat '--highlight-line' option, and bat highlight is using the TextMate theme.

For now I am using syntax highlight group `Visual` background as the bat theme highlight line color.

But I think I can take a look at if I can add 'bold' font style to the highlight line in bat.

**Reply by linrongbin16** [reply]
I think the comment in Feb 16 is not correct, maybe I misunderstood the question.

It looks a good feature, I create an issue for it: https://github.com/linrongbin16/fzfx.nvim/issues/703
